app/src/oculusvr/cpp/DeviceDelegateOculusVR.cpp

```diff
@@ -225,14 +225,13 @@ class OculusLayerSurface: public OculusLayerBase<T, U> {
   OculusLayer::BindDelegate bindDelegate;
 
   void Init(JNIEnv * aEnv, vrb::RenderContextPtr& aContext) override {
-    if (this->swapChain) {
```

Comment:
Does `Init()` get called multiple times? How can the device have a valid swap chain before calling `Init()`?

Reply:
Init can be called from pause/resume life cycle events, and also just after a layer is created when the activity is already active in VR.